browser/src/canvas/sections/ShapeHandlesSection.ts

```diff
@@ -1096,6 +1096,13 @@ class ShapeHandlesSection extends CanvasSectionObject {
 	}
 
 	public onDraw() {
+		this.context.beginPath();
```

Comment:
we could move it to own method to give the name for this piece of a code: drawSelectionFrame or similar, then it's more obvious when reading the code